src/parseTools.js

```diff
@@ -911,14 +911,14 @@ function defineI64Param(name) {
 
 
 function receiveI64ParamAsI53(name, onError, handleErrors = true) {
-  var errorHandler = handleErrors ? `if (isNaN(${name})) return ${onError}` : '';
+  var errorHandler = handleErrors ? `if (isNaN(${name})) { return ${onError}; }` : '';
```

Comment:
Are the extra braces needed here?

Reply:
Not really, I just thought it felt safer in code that's being pasted together. Like, in case `onError` has a semicolon in it or something. Should get minified so it seems OK.